readme.md

```diff
@@ -322,20 +322,21 @@ This also accepts an `object` with the following fields to constrain the duratio
 
 Type: `number | object`<br>
 Default:
-- retries: `2`
+- limit: `2`
+- retryFunction: `(iteration, retryOptions, error, computedValue) => computedValue`
```

Comment:
I think we should use a clearer name here. Maybe `delay()`, `calculateDelay()`, or `getDelay()`. Other suggestions?

Reply:
I'd go with `calculateDelay`.

Reply:
👍